staging/cert-manager-setup/values.yaml

```diff
@@ -82,3 +82,7 @@ clusterissuer: {}
 # This will work only in case when a cert-manager is installed in
 # `cert-manager` namespace.
 installCertManager: true
+
+cert-manager:
+  # This wrapper chart no longer installs CRDs, instead we rely on the upstream chart.
```

Comment:
Does upstream finally offer a way to reliably ensure the CRD is installed and updated to the current version? Or are we just expecting that v1 is mature enough that it's just not going to change?

Reply:
@joejulian I need to check (I hope so). The reason I made the change was that the CRDs no longer fit in a configmap.

```
 Error: UPGRADE FAILED: pre-upgrade hooks failed: warning: Hook pre-upgrade cert-manager-setup/templates/install-crds.yaml failed: ConfigMap "cert-manager-setup-crds" is invalid: []: Too long: must have at most 1048576 bytes
```

```
  File: /home/jared/git/helm/d2iq/charts/staging/cert-manager-setup/files/crds.yaml
  Size: 1679688         Blocks: 3288       IO Block: 4096   regular file
Device: 823h/2083d      Inode: 30045113    Links: 1
Access: (0644/-rw-r--r--)  Uid: ( 1000/   jared)   Gid: ( 1000/   jared)
Access: 2020-07-20 10:41:17.459799075 -0500
Modify: 2020-09-10 08:16:06.290054123 -0500
Change: 2020-09-10 08:16:06.290054123 -0500
 Birth: 2020-07-20 10:41:17.459799075 -0500
```

Reply:
I don't think upstream does now.
They seam to release with Bazel a version which installs CRDs, if `installCRDs` is true.
But haven't seen an upgrade strategy so far.

We may should split our CRDs files as upstream does.
I did this once here https://github.com/mesosphere/charts/commit/2c7263efe91457ffc6ac6cb6df19d16cdda609fb.
https://github.com/jetstack/cert-manager/tree/master/deploy/crds